### PR TITLE
zh-translation: /start/sandboxes/redis.rst

### PR DESCRIPTION
**Pull request by vgbhfive**
For an explanation of how to fill out the fields, please see the relevant section
in [PULL_REQUESTS.md](https://github.com/envoyproxy/envoy/blob/master/PULL_REQUESTS.md)

Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes: /start/sandboxes/redis.rst
Release Notes:
[Optional Runtime guard:]
[Optional Fixes #Issue] #204 
[Optional Deprecated:]

### CODE REVIEW

File: docs/root/start/sandboxes/redis.rst [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
-Running the Sandboxes
-~~~~~~~~~~~~~~~~~~~~~
+在沙盒中运行
+~~~~~~~~~~~~
 
 .. include:: _include/docker-env-setup.rst
 
-Step 3: Build the sandbox
-*************************
+Step 3: 建立沙盒
```
Comment (majinghe) [inline docs/root/start/sandboxes/redis.rst]:
```suggestion
步骤 3： 构建沙盒
```

Comment (majinghe) [reply docs/root/start/sandboxes/redis.rst]: 冒号注意一下，需要中文的。step 翻译过来。

File: docs/root/start/sandboxes/redis.rst [hunk 2]
```diff
@@ -29,12 +29,12 @@ Terminal 1
   redis_proxy_1   /docker-entrypoint.sh /bin       Up      10000/tcp, 0.0.0.0:1999->1999/tcp, 0.0.0.0:8001->8001/tcp
   redis_redis_1   docker-entrypoint.sh redis       Up      0.0.0.0:6379->6379/tcp
 
-Step 4: Issue Redis commands
-****************************
+Step 4: 发出 Redis 命令
```
Comment (majinghe) [inline docs/root/start/sandboxes/redis.rst]:
```suggestion
步骤 4： 发出 Redis 命令
```

File: docs/root/start/sandboxes/redis.rst [hunk 3]
```diff
@@ -47,10 +47,10 @@ Terminal 1
   $ redis-cli -h localhost -p 1999 get bar
   "bar"
 
-Step 5: Verify egress stats
-***************************
+Step 5: 验证出口统计
```
Comment (majinghe) [inline docs/root/start/sandboxes/redis.rst]:
```suggestion
步骤 5： 验证出口统计
```

File: docs/root/start/sandboxes/redis.rst [hunk 4]
```diff
@@ -1,20 +1,20 @@
...
+Redis 过滤器
 ============
 
-In this example, we show how a :ref:`Redis filter <config_network_filters_redis_proxy>` can be used with the Envoy proxy. The Envoy proxy configuration includes a Redis filter that routes egress requests to redis server.
+在这个示例中，我们会展示如何将 :ref:`Redis 过滤器 <config_network_filters_redis_proxy>` 和 Envoy 代理一起使用。Envoy 代理配置包含一个 Redis 过滤器，它会将出口请求路由到 Redis 服务器。
 
-
-Running the Sandboxes
-~~~~~~~~~~~~~~~~~~~~~
+在沙盒中运行
```
Comment (majinghe) [inline docs/root/start/sandboxes/redis.rst]:
```suggestion
运行沙盒
```

**Comment by majinghe** [comment]
@vgbhfive  加了些 comment。处理一下。

**Comment by vgbhfive** [comment]
> @vgbhfive 加了些 comment。处理一下。

感谢纠正，已处理。

**Comment by helight** [comment]
LGTM
/approve

**Pull request merged**